src/nfts/NftsInfoCarousel.test.tsx

```diff
@@ -124,4 +133,40 @@ describe('NftsInfoCarousel', () => {
 
     expect(queryByTestId('ViewOnExplorer')).toBeNull()
   })
+
+  it('getGatewayUrl', () => {
+    expect(getGatewayUrl(mockNftAllFields)).toEqual(mockNftAllFields.media[0].gateway)
+  })
+
+  it('onImageLoad success', () => {
+    onImageLoad(mockNftAllFields, NftOrigin.TransactionFeed, false)
+    expect(ValoraAnalytics.track).toBeCalledWith('nft_image_load', {
+      tokenId: mockNftAllFields.tokenId,
+      contractAddress: mockNftAllFields.contractAddress,
+      url: mockNftAllFields.media[0].gateway,
+      origin: NftOrigin.TransactionFeed,
+      error: false,
+    })
+    expect(ValoraAnalytics.track).toBeCalledTimes(1)
+    expect(Logger.info).toBeCalledWith(
+      NftOrigin.TransactionFeed,
+      `ContractAddress=${mockNftAllFields.contractAddress}, TokenId: ${mockNftAllFields.tokenId}, Loaded image from ${mockNftAllFields.media[0].gateway}`
+    )
+  })
+
+  it('onImageLoad error', () => {
```

Comment:
nit: can we give these dedicated `describe` blocks and have a more meaningful `it` strings (verb + description)?

Reply:
This is just a temporary place for these helper functions. They will be moved in a follow up PR.